packages/models/src/purpose/protobufConverterFromV2.ts

```diff
@@ -82,7 +82,9 @@ export const fromPurposeV2 = (input: PurposeV2): Purpose => ({
   eserviceId: unsafeBrandId(input.eserviceId),
   consumerId: unsafeBrandId(input.consumerId),
   versions: input.versions.map(fromPurposeVersionV2),
-  isFreeOfCharge: input.isFreeOfCharge,
+  freeOfChargeReason: input.isFreeOfCharge
+    ? input.freeOfChargeReason
+    : "Sono una Pubblica Amministrazione",
```

Comment:
Same as above, for consistency.

Reply:
not sure if we should handle this default in the converter. This could be considered part of the creation/update business logic, and it could probably be only there.

(it's ok for the v1, because we needed to handle existing data)

@beetlecrunch thoughts?